### ISSUE TITLE
[css-env-1] defining variables

### ISSUE THREAD

**Issue by ghost** [w3c/csswg-drafts#2629]
This is a follow up to: https://drafts.csswg.org/css-env-1/#issue-ed0904e4

The current Safari implementation does not have a way to define variables. A noticeable difference from var is the missing double dash at the beginning of variable names e.g. env(safe-area-inset-let). Maybe we could prefix developer defined variables with the double dash and UA defined variables should not have the double dash.

This would make it easier to differentiate between the two types of variables, would stop the developer overriding UA variables and ensure consistency with what has already been shipped.

**Comment by tabatkins** [comment]
Yeah, that's the intent - author-defined ones would be `--foo`, so they live in a separate namespace from UA-defined ones, and we don't have to worry about clashes.

**Comment by bramus** [comment]
Closing this one in favor of https://github.com/w3c/csswg-drafts/issues/2627 which discusses potential syntaxes to define custom envvars.
